modules/installation/proc-installing-with-helm-cli.adoc

```diff
@@ -0,0 +1,57 @@
+[id="installing-with-helm-cli_{context}"]
+= Installing {product} with Helm CLI
+
+As an OpenShift or Kubernetes administrator, you can use the Helm CLI to install {product}.
+
+.Prerequisites
+* An `oc` or `kubectl` session with a user that has `admin` role on an OpenShift or Kubernetes cluster.
+* Helm CLI.
+
+.Procedure
+. Add the OpenShift Helm Charts repository:
++
+----
+$ helm repo add openshift-helm-charts https://charts.openshift.io/
+----
+
+. Create and activate the _<rhdh>_ OpenShift project or Kubernetes namespace:
++
+[subs="quotes+"]
+----
+$ NAMESPACE=_<rhdh>_
+$ oc new-project ${NAMESPACE}
+$ oc project ${NAMESPACE}
+----
+
+. Install the {product-short} Helm Chart:
++
+[subs="attributes+"]
+----
+$ helm upgrade redhat-developer-hub -i https://github.com/openshift-helm-charts/charts/releases/download/redhat-redhat-developer-hub-{product-chart-version}/redhat-developer-hub-{product-chart-version}.tgz
```

Comment:
note that this version was wrong in 1.2 and main branches. See https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/288 and https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/289